internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -804,7 +804,9 @@ func convertPublish(topics []manifest.Topic, accountID, region, app, env, svc st
 	// convert the topics to template Topics
 	for _, topic := range topics {
 		var fifoConfig *template.FIFOTopicConfig
+		topicName := aws.StringValue(topic.Name)
 		if topic.FIFO.IsEnabled() {
+			topicName += ".fifo"
```

Comment:
So we only had to fix for SNS but not SQS right because we have:
https://github.com/aws/copilot-cli/blob/dd8dc3356ef47130587fb6092e7a29a716899f45/internal/pkg/manifest/worker_svc.go#L245-L258

Can we not do a similar thing for SNS? like updating the existing `Publish` method to handle this logic. Otherwise, we're being inconsistent with where this augmentation happens

Reply:
I totally agree to your point. I have changed the implementation, do let me know how do you feel about new implementation.

Reply:
I think I will need to make the similar change for the Job as well. I have done it for all the service types. I will do it for the job as well in a bit.